build.gradle

```diff
@@ -5,7 +5,7 @@ plugins {
 }
 
 group = 'com.glencoesoftware.omero'
-version = '0.9.2-SNAPSHOT'
```

Comment:
Was this change for testing locally?

Reply:
Yes, totally up for incrementing the minor version number for this. Could we make it `0.10.0-SNAPSHOT` for now so that we can aim to merge this without necessarily calling it a full release? 

Looking into #26 as it might be timely to move these API methods upstream.